src/App.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div id="app">
+    <h1 class="title">  The list of Addons :</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>
+            Name
+          </th>
```

Comment:
I did a test, when I hover the table headers, an hand icon is displayed which made me think I could sort the table, but when I clicked nothing happened. We should either remove this icon, either make the table sortable.

Reply:
done